languages/emf-metamodel/src/main/java/de/jplag/emf/util/AbstractModelView.java

```diff
@@ -0,0 +1,43 @@
+package de.jplag.emf.util;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Textual representation of a model for the depiction of matches in submissions.
+ * @author Timur Saglam
+ */
+public class AbstractModelView {
```

Comment:
Why is this class specific to models? Looks to me rather like a utility class which is pretty generic for writing any string to a file? Sure there is not already some functionality in the Java API for this?

Reply:
I would have expected some method like `FileUtils.write(File, String)` but seems there is none in the stdlib.